### DISCUSSION TITLE
"You can't see this item" on my Project

### DISCUSSION THREAD

**Discussion by ayo-run** [community/community#36409] [Projects and Issues]
<img width="472" alt="Screen Shot 2022-10-17 at 1 34 20 PM" src="https://user-images.githubusercontent.com/4262489/196167213-3771cec0-b1e4-4182-b980-6e35105f0a50.png">

What could this possibly be, and how can I remove it? I can't open it. 🥲

**Comment by jfullstackdev** [comment]
I'm seeing the same thing 

![image](https://user-images.githubusercontent.com/47092464/196223504-cdd4a493-b70b-441c-bd52-7f48f3172059.png)

would that be related to private repos ? maybe you're not seeing them when not logged-in

**Reply by ayo-run** [reply]
I'm logged in and I can see all my private repository issues... Is it possible for others to add an item on my public project?

**Reply by jfullstackdev** [reply]
maybe using of other services in your projects ? bots ? not very sure of that

**Reply by jfullstackdev** [reply]
I'll also link the other post once it's confirmed it's the same problem of not seeing the item

**Reply by khacminh** [reply]
The issue occurs when the user doesn't have permission to access the parent repo of the issue. Unfortunately, only enterprise subscription organizations can create fine-grained permission that allows users to read/create issues without access to the source code.
Another (silly) workaround is creating another issue-only repo beside the private source code repo.

I don't know why GitHub doesn't have static **reporter** permission.

**Comment by ossanna16** [comment]
Hey there @ayoayco and thank you for asking a good question! I'm glad you found our community 🙂

**Comment by tim-bezhashvyly** [comment]
For me this happened for the repo I have deleted. So its ticket has been assigned to the board and perfectly visible but after the deletion turned into "You can't see this item". No idea how to remove it.

**Comment by md84419** [comment]
I've seen this today with a project to which I've been added.  I'm in a different organisation to the project owner.

The workaround is to put the following into the filter: "is:open,closed".

Also see #8550. As @dustingoodman has suggested, maybe tickets can have an internal state (such as deleted or orphaned) which isn't exposed through the filter as a TYPE or STATE, and the filter logic isn't filtering them out correctly? https://docs.github.com/en/issues/planning-and-tracking-with-projects/customizing-views-in-your-project/filtering-projects#filtering-for-item-state-or-item-type

**Reply by tim-bezhashvyly** [reply]
Cool. You can also save the filter so it stays default. Thank you for a workaround.
